nestkernel/nest_names.cpp

```diff
@@ -364,6 +367,9 @@ const Name withgid( "withgid" );
 const Name withpath( "withpath" );
 const Name withtime( "withtime" );
 const Name withweight( "withweight" );
+const Name withreceivergid( "withreceivergid" );
```

Comment:
Would not `withtarget` be a much better name (also for pertaining variable names in the code)?